build.gradle

```diff
@@ -1710,6 +1710,7 @@ project(':streams') {
     testImplementation libs.powermockEasymock
```

Comment:
Can we remove the powermock dependencies or are there other tests in Streams using it?

Reply:
There are other dependencies that have not been modified, so this cannot be removed .